wp-hooks-docs/src/theme/Footer/index.js

```diff
@@ -0,0 +1,162 @@
+ */
+function GetStartedSection() {
+	return (
+		<section
+			className={ clsx( styles.getStartedSection, styles.greySection ) }
+		>
+			<div className="container">
+				<h2 className={ styles.sectionTitle }>Ready to Get Started?</h2>
+				<p className={ styles.sectionDescription }>
+					Distributor is a WordPress plugin that makes it easy to
+					syndicate and reuse content across your websites — whether
+					in a single multisite or across the web.
+				</p>
+				<div className={ styles.getStartedActions }>
+					<Link
+						className="button button--primary button--lg"
+						to="https://distributorplugin.com/#cta"
+					>
+						Download Distributor Plugin
+					</Link>
+				</div>
+			</div>
+		</section>
+	);
+}
+
+function Footer() {
+	const currentYear = new Date().getFullYear().toString();
+
+	return (
+		<>
+			<GetStartedSection />
+			<footer
+				className={ clsx(
+					ThemeClassNames.layout.footer.container,
+					styles.mainFooter,
+					'footer'
+				) }
+			>
+				<span
+					aria-hidden="true"
+					className={ styles.footerBackground }
+				></span>
+				<div className="container">
+					<div className={ styles.footerContainer }>
+						<div className={ styles.footerCopyright }>
+							<div className={ styles.footerCopyrightLinks }>
+								<FooterCopyright
+									copyright={ `Finely crafted by Fueled, ©${ currentYear }` }
+								/>
+								<a href="https://github.com/10up/distributor/issues/">
+									Support
+								</a>
+								&nbsp;
+								<span>&nbsp;|&nbsp;</span>
+								<a href="https://github.com/10up/distributor/">
+									Contribute
+								</a>
+								&nbsp;
+								<span>&nbsp;|&nbsp;</span>
+								<a href="https://distributorplugin.com/privacy-policy/">
+									Privacy Policy
+								</a>
+							</div>
+						</div>
+						<div className={ styles.footerLogo }>
+							<img
+								loading="lazy"
+								decoding="async"
+								width="172"
+								height="33"
+								src="https://distributorplugin.com/wp-content/uploads/sites/2/2025/06/lockup_white.svg"
+								alt="Fueled Logo"
+							/>
+							<div className={ styles.footer10upButton }>
+								<div className={ styles.footer10upButtonInner }>
+									<a
+										className="wp-block-button__link wp-element-button"
+										href="https://fueled.com/wordpress"
+									>
+										Engineered by the{ ' ' }
+										<img
+											loading="lazy"
+											decoding="async"
+											width="21"
+											height="24"
+											style={ { width: '21px' } }
+											src="https://distributorplugin.com/wp-content/uploads/sites/2/2025/06/svgexport-3-1.svg"
+											alt="10up Logo"
+										/>{ ' ' }
+										WordPress Practice
+									</a>
+								</div>
+							</div>
+						</div>
+						<div className={ styles.footerSocial }>
+							<a
+								href="https://www.facebook.com/10up.agency/?ref=distributor"
+								title="10up on Facebook"
+								aria-hidden="true"
```

Comment:
At the moment these don't appear in the accessibility tree (checked with firefox dev tools)

```suggestion
								aria-label="10up on Facebook"
```

:1234: Applies to the other social links too